modules/ocf_desktop/files/xsession/Xsession

```diff
@@ -118,3 +118,7 @@ sleep 8 && /usr/local/bin/fix-audio &
 
 # Source custom rc shared across desktops
 [ -f ~/remote/.desktoprc ] && . ~/remote/.desktoprc
+
+# Enable compositor
+xfconf-query -c xfwm4 -p /general/use_compositing -s false
```

Comment:
don't we already set it to false?
https://github.com/ocf/puppet/blob/d0b998a059bbdcd07b0c1433f53cff87438f97b7/modules/ocf_desktop/files/xsession/xfce4/xfconf/xfce-perchannel-xml/xfwm4.xml#L67

is that config file not working?

FWIW i think we should set it in a config file, not control it in a script at login time

Reply:
side note: if we really *aren't* using a compositor now, have we tried just turning on whatever the xfce default one and seeing if that's sufficient? less maintenance for us.

Reply:
No, we aren't using a compositor now; the config file works just fine. I just put that line in to make it clear that two compositors aren't running at the same time, so it's not really necessary.

What really pissed me off was when I'd drag a terminal somewhat fast and there would be a delay with 10 of them following it. Essentially, it looks like the few moments before your Windows 95 system crashes. Both the xfce compositor and compton fix this issue. But, for example, when running a [tearing test](https://www.youtube.com/watch?v=9hIRq5HTh5s), the xfce compositor still has noticeable tearing whereas compton does not. @tliu22 can vouch for this too.

I'd like if someone more well-versed in the graphics realm, such as @harrisonkuo, shared their opinion on this too.